CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## UNRELEASED
+### Updated
+- Upgraded plotly.js to 1.44.3 [#458](https://github.com/plotly/dash-core-components/pull/458)
+  - [Patch release 1.44.3](https://github.com/plotly/plotly.js/releases/tag/v1.44.3) which contains:
+    - Fixed axis `automargin` push offset
+    - Fix handling of alpha channel in rgba coloring in `scatter3d` traces
+    - Fixed subplots with multiple `carpet` traces
+    - Fixed MathJax placement in ternary `aaxis` titles
```

Comment:
Alright. Cool. Now for 1.44.2 :)

Reply:
@cldougl apologies, this is a recent change but I don't want to include all the details of the plotly.js releases in the dcc changelog - just the highlights for new features, and no extra info for bugfixes. Otherwise the dcc changelog gets swamped by plotly.js features. You can just use exactly the text @Marc-Andre-Rivet wrote above.

Reply:
no problem! let me update that